composer.json

```diff
@@ -24,8 +24,8 @@
         "ext-soap": "*",
         "ext-curl": "*",
         "ass/xmlsecurity": "~1.0",
-        "symfony/framework-bundle": "~2.6",
-        "symfony/twig-bundle": "~2.6",
+        "symfony/framework-bundle": "~2.0|~3.0|~4.0",
+        "symfony/twig-bundle": "~2.0|~3.0|~4.0",
```

Comment:
I think we can just keep it at `^3.4|^4.0`, for the purpose of this fork, even `^4.0` could be sufficient.